static/scripts/javascript.js

```diff
@@ -539,6 +539,28 @@ var app = {
         var el = e.target.closest(".js-results-container");
         el.setAttribute("data-fetch-url", e.target.value);
         el.dispatchEvent(new Event("update"));
+        if (e.target.id === "find-race") {
+          $("#find-candidate")
+            .val("1")
+            .trigger("change");
+          $("#find-district")
+            .val("1")
+            .trigger("change");
+        } else if (e.target.id === "find-candidate") {
+          $("#find-race")
+            .val("1")
+            .trigger("change");
+          $("#find-district")
+            .val("1")
+            .trigger("change");
+        } else {
+          $("#find-race")
+            .val("1")
+            .trigger("change");
+          $("#find-candidate")
+            .val("1")
+            .trigger("change");
+        }
```

Comment:
This will change them all but then fix the one we don't want to change.

```js
var tempVal = $(e.target).val();
$('.js-select2').val(null).trigger('change');
$(e.target).val(tempVal).trigger('change');
```

Reply:
good deal -- i'll just make the val match the first option so we still have "choose by x" on change

Reply:
Ah, their advice was probably assuming that there wasn't a default value.